yt_dlp/extractor/nubilesporn.py

```diff
@@ -0,0 +1,143 @@
+                {'url': _VALID_VIDEO_URL, 'format_id': 'mp4_960_540', 'width': 960, 'height': 540},
+                {'url': _VALID_VIDEO_URL, 'format_id': 'mp4_1280_720', 'width': 1280, 'height': 720},
+                {'url': _VALID_VIDEO_URL, 'format_id': 'mp4_1920_1080', 'width': 1920, 'height': 1080},
+                {'url': _VALID_VIDEO_URL, 'format_id': 'mp4_3840_2160', 'width': 3840, 'height': 2160}
+            ],
+            'ext': 'mp4',
+            'display_id': 'trying-to-focus-my-one-track-mind-s3e1',
+            'thumbnail': f'https://images.{_BASE_DOMAIN}/videos/trying_to_focus_my_one_track_mind/samples/cover1280.jpg',
+            'description': 'md5:81f3d4372e0e39bff5c801da277a5141',
+            'creator': 'NubilesPorn',
+            'release_timestamp': 1676160000,
+            'release_date': '20230212',
+            'uploader': 'NubilesPorn',
+            'uploader_url': _MEMBERS_URL,
+            'timestamp': 1676160000,
+            'upload_date': '20230212',
+            'channel': 'Younger Mommy',
+            'channel_id': 64,
+            'channel_url': f'{_MEMBERS_URL}/video/website/64',
+            'like_count': int,
+            'average_rating': float,
+            'age_limit': 18,
+            'webpage_url': f'{_MEMBERS_URL}/video/watch/165320/trying-to-focus-my-one-track-mind-s3e1',
+            'categories': ['Big Boobs', 'Big Naturals', 'Blowjob', 'Brunette', 'Cowgirl', 'Girl Orgasm', 'Girl-Boy',
+                           'Glasses', 'Hardcore', 'Milf', 'Shaved Pussy', 'Tattoos', 'YoungerMommy.com'],
+            'tags': list,
+            'cast': ['Kenzie Love'],
+            'availability': 'needs_auth',
+            'series': 'Younger Mommy',
+            'series_id': 64,
+            'season': 'Season 3',
+            'season_number': 3,
+            'episode': 'Episode 1',
+            'episode_number': 1
+        }
+    }]
+
+    def _perform_login(self, username, password):
+        login_webpage: str = self._download_webpage(f'{NubilesPornIE._BASE_URL}/login', video_id=None)
+        hidden_inputs: dict = self._hidden_inputs(login_webpage)
+        hidden_inputs.update({'username': username, 'password': password})
+        self._request_webpage(f'{NubilesPornIE._BASE_URL}/authentication/login',
+                              data=urlencode_postdata(hidden_inputs), video_id=None)
+
+    @staticmethod
+    def _get_channel_info(element: str) -> dict:
+        info = {}
+        path = extract_attributes(get_element_html_by_class('site-link', element)).get('href')
+        if path:
+            info['url'] = f'{NubilesPornIE._MEMBERS_URL}{path}'
+            info['id'] = int_or_none(re.findall('/([0-9]+)$', path)[0])
+        raw_name = get_element_by_class('site-link', element)
+        if raw_name:
+            info['name'] = ' '.join(re.findall('[A-Z][^A-Z]*', raw_name.replace('.com', '')))
+        return info
```

Comment:
Why put it in an intermediate dict, only to unpack it later? Extracting in _real_extract directly would make it way simpler